internal/hns/hnsnetwork.go

```diff
@@ -51,45 +54,184 @@ type hnsResponse struct {
 	Output  json.RawMessage
 }
 
+func CheckForErrors(hr int, result string) string {
```

Comment:
Why not convert this to an error to be more `go-ish`

Reply:
Also why is it exported? I dont see any users outside this file?

Reply:
Once hnsnetwork is tested and working, I'll have similar changes for hnsendpoint and hnsnamespace. This could get moved somewhere more central.